applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/formfinding_strategy.hpp

```diff
@@ -198,7 +199,7 @@ class FormfindingStrategy
         if (mPrintingFormat=="all" || mPrintingFormat=="gid"){
             mpIterationIO = Kratos::make_unique<IterationIOType>(
                 "formfinding_iterations",
-                GiD_PostAscii, // GiD_PostAscii // for debugging GiD_PostBinary
+                GiD_PostAscii,
```

Comment:
```suggestion
                GiD_PostBinary,
```

ascii is only for debugging

Reply:
I cant open GID binaries on my machine anymore, so .... :D

Reply:
But then you should fix your GiD I propose ...

Reply:
I commited your suggestion. Anyway, also installing new GID versions does not fix my problem....